packages/pg/lib/logger.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
```

Comment:
Why is this file needed?

Reply:
It sets up the logger: setting a default log level(`info`) and allowing users to change the log level either by `LOG_LEVEL` enviornment variable or `setLogLevel()` method.